packages/block-editor/src/components/dimensions-tool/scale-tool.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalToolsPanelItem as ToolsPanelItem,
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+} from '@wordpress/components';
+import { useMemo } from '@wordpress/element';
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * @typedef {import('@wordpress/components/build-types/select-control/types').SelectControlProps} SelectControlProps
+ */
+
+/**
+ * @type {SelectControlProps[]}
+ */
+const DEFAULT_SCALE_OPTIONS = [
+	{
+		value: 'cover',
+		label: _x( 'Cover', 'Scale option for dimensions control' ),
+		help: __(
+			'Content is scaled and cropped to fill the entire space without being distorted.'
+		),
+	},
+	{
+		value: 'contain',
+		label: _x( 'Contain', 'Scale option for dimensions control' ),
+		help: __(
+			'Content is scaled to fill the space without clipping nor distorting.'
```

Comment:
```suggestion
			'Content is scaled to fill the space without clipping or distorting.'
```

Reply:
This was copied over from the post featured image block. Eventually I'd like to replace that component with this one, so I'm not going to bother changing it over there for now.